e2e/support/commands/database/addH2SampleDatabase.js

```diff
@@ -1,9 +1,6 @@
 Cypress.Commands.add(
   "addH2SampleDatabase",
   ({ name, auto_run_queries = false, is_full_sync = false } = {}) => {
-    // IMPORTANT!
-    // TODO: Remove the following line when https://github.com/metabase/metabase/issues/24900 gets fixed.
-    cy.skipOn(true);
```

Comment:
I use this function in the test to add a new H2 database, so I need to remove this line (since the issue was fixed)